api/envoy/extensions/wasm/v3/service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
```

Comment:
This should probably stay in the original file (`api/envoy/extensions/wasm/v3/wasm.proto`) to follow naming of other extensions.